vng_api_common/notifications/handlers.py

```diff
@@ -1,5 +1,14 @@
 import logging
 
+from django.conf import settings
+from django.utils.module_loading import import_string
+
+from djangorestframework_camel_case.util import underscoreize
+
+from vng_api_common.authorizations.models import Applicatie
```

Comment:
can you use relative imports instead? I'm not putting it past people to want to rename this thing again... 
```suggestion
from ..authorizations.models import Applicatie
```